archimedes/archimedes/functions/plotting/__init__.py

```diff
@@ -1,2 +1,4 @@
 import plotly.express as px
 import plotly.io as pio
```

Comment:
Wonder if it would it be easier for someone to jump into scripting for Vulcan if we leave these called plotly.express and plotly.io here?

Means the `import plotly.express as px` would be in the downstream scripts, but I think the immediate recognizability of "Oh, `px` is plotly.express" saves a new coder from needing to either dive into our backend system from the get-go or else ask us what those are.

Reply:
I think that's from my hacking around to work on the UI earlier. I definitely think it's a good idea to rename these to whatever makes more sense.

Reply:
Hmm, alternatively: do we want to actually expose all of plotly.express functionality to the workflow writer, or do we want to give them the `scatter` function with a defined interface, which may or may not use plotly.express under the hood?

Reply:
God question!  I think both? I expect sometimes a workflow might use packages with their own visualization functions (like scanpy has, though we're ignoring them) and it's be nice to be able to at least let people start with those (perhaps staying that way forever, or perhaps someone *cough, me* would be in charge of re-implementing on our viz-tooling). In such cases, it's probably good to have the raw tooling available for any necessary tweaks.